src/agentscope/agents/__init__.py

```diff
@@ -16,7 +15,6 @@
     "Operator",
     "DialogAgent",
     "DictDialogAgent",
-    "TextToImageAgent",
```

Comment:
- Delete related tutorial in [201-agent.md](https://github.com/modelscope/agentscope/blob/main/docs/sphinx_doc/en/source/tutorial/201-agent.md?plain=1#L73)
- Remove related code in both Workstation Node and Studio html-drag-components

Reply:
Maybe we should transform the `TextToImageAgent` in workstation into `dashscope_text_to_image` service while maintaining the same interface and functionality since Text to Image is interesting and attractive.

Reply:
> * Delete related tutorial in [201-agent.md](https://github.com/modelscope/agentscope/blob/main/docs/sphinx_doc/en/source/tutorial/201-agent.md?plain=1#L73)
> * Remove related code in both Workstation Node and Studio html-drag-components

Modified accordingly.

Reply:
> Maybe we should transform the `TextToImageAgent` in workstation into `dashscope_text_to_image` service while maintaining the same interface and functionality since Text to Image is interesting and attractive.

Remove code related to text-to-image agent in workstation for now. Later a text-to-image service function should be added.